openmc/data/angle_distribution.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, energy, mu):
         self.energy = energy
         self.mu = mu
 
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return NotImplemented
```

Comment:
Why not return False? Seems like if other is not even the same type of object, how could it be equal?